frontend/src/components/Post/Post.js

```diff
@@ -126,8 +126,8 @@ function Post({
         uris: [uri],
         // position: 0,
       };
-      console.log("data: ", data);
-      console.log("id: ", id);
+      // console.log("data: ", data);
+      // console.log("id: ", id);
```

Comment:
get rid of console logs

Reply:
Totally forgot about those, but they should all be gone now.